database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/NewPostsView.swift

```diff
@@ -24,6 +24,11 @@ struct NewPostsView: View {
 
   var body: some View {
     HStack {
+      Button(action: {
+        isPresented = false
+      }) {
+        Image(systemName: "chevron.left")
+      }
```

Comment:
Double checking, was this new Button view intended to appear on all platforms or just tvOS?

Reply:
the new button appears on all platforms